cli/gardener_ci/_cosign.py

```diff
@@ -89,7 +90,9 @@ def sign(
     cosign.sign_image(
         image_reference=image_reference,
         signature_image_reference=signature_image_ref,
-        signature=signature.signature,
-        on_exist=cosign.OnExist.OVERWRITE,
+        signature=signature.signature.replace('\n', ''),
+        signing_algorithm=signing_algorithm,
+        public_key=signature.public_key.replace('\n', ''),
+        on_exist=cosign.OnExist(on_exist),
```

Comment:
Parsing to enum could be omitted here since `sign_image` also accepts string values and checks if it is part of the enum.